lib/utils/UIConstants.dart

```diff
@@ -0,0 +1,32 @@
+import 'dart:ui';
+
+abstract class UIConstants {
```

Comment:
keep the convention of the file names equal, lower case separated with underscores

Reply:
fixed